cypress/e2e/default/learn/common-components/navbar.ts

```diff
@@ -0,0 +1,85 @@
+const navBarselectors = {
+  heading: "[data-test-label='landing-header']",
+  smallCallToAction: "[data-test-label='landing-small-cta']",
+  navigationLinks: '.nav-list',
+  avatarContainer: '.avatar-container',
+  defaultAvatar: '.avatar-container',
+  menuButton: '.toggle-button-nav',
+  avatarImage: '.avatar-container .avatar'
+};
+
+
+
+
```

Comment:
```suggestion

```

Reply:
That step was completed two days ago but I never managed to push it.